core/src/main/java/org/apache/spark/SparkFirehoseListener.java

```diff
@@ -30,6 +30,14 @@
  */
 public class SparkFirehoseListener implements SparkListenerInterface {
 
+  @Override
+  public boolean dead() {
+    return false;
+  }
+
+  @Override
+  public void dead_$eq(boolean dead) { }
```

Comment:
these methods should actually be implemented, so anybody extending this in java gets the fix as well.

ideally we could do this somehow so this doesn't get exposed at all part of the api, but I can't think of a way to do that ...